client/src/components/molecules/ChatInput.tsx

```diff
@@ -1,7 +1,7 @@
 import React, { useCallback, useMemo } from "react";
 import { css } from "@emotion/react";
-
-import { EmoticonIcon, SendIcon } from "@/assets";
+// import { EmoticonIcon, SendIcon } from "@/assets";
+import { SendIcon } from "@/assets"; // remove emoji
```

Comment:
코드에 대해서 설명을 남기고 싶을 때는 코드 내에 주석으로 남기는 것보다 깃헙 PR 내에서 자신의 작업사항에 대해서도 댓글을 남길 수 있으니 그렇게 남겨주시면 될 것 같아요!

```suggestion
import { SendIcon } from "@/assets";
```